wled00/button.cpp

```diff
@@ -25,11 +25,13 @@ void shortPressAction(uint8_t b)
   }
 
   // publish MQTT message
+  #ifndef WLED_DISABLE_MQTT
   if (buttonPublishMqtt && WLED_MQTT_CONNECTED) {
     char subuf[64];
     sprintf_P(subuf, _mqtt_topic_button, mqttDeviceTopic, (int)b);
     mqtt->publish(subuf, 0, false, "short");
   }
+  #endif
```

Comment:
this pattern is repeated a lot in this file, consider writing a helper functions.